js/client-teambuilder.js

```diff
@@ -109,6 +109,7 @@
 		update: function () {
 			teams = Storage.teams;
 			if (this.curTeam) {
+				this.ignoreEVLimits = (this.curTeam.gen < 3) || (this.curTeam.format === 'classichackmons');
```

Comment:
Ah, so no teambuilder formats then. I've updated the branch, assuming `git rebase` hasn't destroyed anything again.